nss-day-cohort-60/national-parks-api
# Jd refactor amenity and user views

### Description
Refactored get_all* and get_single* functions in natural_attraction_views, user_views, and amenity_views to utilize get_all() and get_single() functions from sql_helper to streamline code. 
Changed spelling to "campground" from "campround" on 2 lines in national_park_create_database.sql file

### Steps to test get_all()
Fetch this brach and switch to it
Start/restart debugger
Send a GET request for amenity_types, users, and natural_attractions (e.g. http://localhost:8088/campgrounds)
Verify that 200 status code and the entire list of dictionaries are received

### Steps to test get_single()
Send a GET request for an existing id from amenity_types, users, and natural_attractions (e.g. http://localhost:8088/campgrounds/1)
Verify that 200 status code and the single dictionary with id = 1 are received
